test/e2e/apps/deployment.go

```diff
@@ -880,12 +884,15 @@ func testProportionalScalingDeployment(f *framework.Framework) {
 	err = framework.WaitForReplicaSetDesiredReplicas(c.ExtensionsV1beta1(), secondRS)
 	Expect(err).NotTo(HaveOccurred())
 
-	framework.Logf("Verifying that first rollout's replicaset has 11 available replicas")
-	err = framework.WaitForReplicaSetAvailableReplicas(c.ExtensionsV1beta1(), firstRS, 11)
+	biggerProportion := (newReplicas - replicas + 1) / 2
```

Comment:
`With proportional scaling, we spread the additional replicas across all ReplicaSets. Bigger proportions go to the ReplicaSets with the most replicas and lower proportions go to ReplicaSets with less replicas. Any leftovers are added to the ReplicaSet with the most replicas. ReplicaSets with zero replicas are not scaled up.`

To clarify the formula, we have 2 ReplicaSets, thus `/ 2`. Note that `+ 1` is necessary to round up for bigger proportion.